src/Features/Core/Portable/CodeStyle/AbstractCodeStyleDiagnosticAnalyzer.cs

```diff
@@ -12,13 +12,7 @@ internal abstract class AbstractCodeStyleDiagnosticAnalyzer : DiagnosticAnalyzer
     {
         protected readonly string DescriptorId;
 
-        /// <summary>
-        /// Diagnostic descriptors corresponding to each of the DiagnosticSeverities.
-        /// </summary>
-        protected readonly DiagnosticDescriptor HiddenDescriptor;
-        protected readonly DiagnosticDescriptor InfoDescriptor;
-        protected readonly DiagnosticDescriptor WarningDescriptor;
-        protected readonly DiagnosticDescriptor ErrorDescriptor;
```

Comment:
I recollect that this was added way back when we did not have `OpenFileOnly` flag and we decided closed file analysis based on whether the analyzer reports any non-hidden diagnostics. @sharwell, you have a PR that remove `OpenFileOnly` flag and moves us back to the latter approach correct? How will address this issue when both your PRs are put together?

Reply:
➡️ I have no plans to move the other pull request forward in its current form. I'll make any necessary adjustments at the time it is revisited.